src/lib/components/AccessRightField.js

```diff
@@ -111,13 +121,47 @@ export class AccessRightField extends Component {
   }
 }
 
-AccessRightField.propTypes = {
+class FormikAccessRightField extends Component {
```

Comment:
This can be renamed to just `AccessRrightField` same fashion as the other components...

Reply:
We can't here because of the double-wrapping:

Raw component: `AccessRightFieldComponent ->` Formik wrap: `FormikAccessRightField->` Redux wrap and Final name: `AccessRightField`